src/components/Form.js

```diff
@@ -51,57 +53,58 @@ export default function Form({ handleData }) {
           {...register('foodName', {
             required: {
               value: true,
-              message: 'Wie willst Du etwas wiederfinden ohne Namen?',
+              message:
+                'How do you want to find something again without a name?',
             },
             maxLength: {
               value: 20,
-              message: 'ups, Limit von 2o Buchstaben erreicht',
+              message: 'ups, limit of 2o letters reached',
             },
             pattern: {
-              value: /^[A-Za-z]+/i,
-              message: 'bitte Text eingeben',
+              value: /^[A-Za-z ]+$/,
+              message: 'please enter text',
```

Comment:
Maybe you could write here which characters are allowed in the error message e.g. "Please use only ..." or "No numbers or special characters allowed"

Reply:
thank you, i grabbed your last suggestion :-)